components/graphs/Graph.tsx

```diff
@@ -56,21 +72,26 @@ const Graph: FC<{ siteUUID: string }> = ({ siteUUID }) => {
     updateEnabled: timeEnabled,
   });
 
+  const [timeRange, setTimeRange] = useState('24');
+  const handleChange = (event: any) => {
+    setTimeRange(event.target.value);
```

Comment:
Do the parseInt here instead
```suggestion
    setTimeRange(parseInt(event.target.value));
```